modules/bitgo/test/v2/unit/coins/utxo/recovery/backupKeyRecovery.ts

```diff
@@ -229,6 +229,10 @@ function run(
 }
 
 utxoCoins.forEach((coin) => {
+  if (coin.getFamily() !== 'doge') {
```

Comment:
Was this intentional or only for testing?

Reply:
oh no, only for testing 🤦